src/Scribunto/ScribuntoLuaLibrary.php

```diff
@@ -33,6 +33,28 @@ public function register(): void {
 		$this->getEngine()->registerInterface( __DIR__ . '/' . 'mw.wsslots.lua', $interfaceFuncs, [] );
 	}
 
+	/**
+	 * @param \WikiPage $wikiPage
+	 *
+	 * @return bool
+	 */
+	private function userCan( \WikiPage $wikiPage ): bool {
+		// Only do a check for user rights when not in cli mode
+		if ( PHP_SAPI != 'cli' ) {
```

Comment:
Reverse the `if` statement expression and use a guard-clause instead. That is, have the entire `if` block be:

```php
if ( PHP_SAPI === 'cli' ) {
    return true;
}
```